src/Logic/operate.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
```

Comment:
- [ ] Division by zero is not being handled, thus, causing an app crash, please handle it accordingly to comply with the project requirements and to avoid the app from crashing:

![image](https://user-images.githubusercontent.com/38522578/138267670-0a5578de-f9e0-4c71-8d63-a7a516885763.png)